### DISCUSSION TITLE
Autograding bot is not grading

### DISCUSSION THREAD

**Discussion by Shahmeer-Nasir** [skills/.github#343] [Introduction to GitHub]
On my first try I named new branch "first-branch" instead of "my-first-branch". The page won't move to next step. Anyway I found out about naming it exactly the same in the discussions. After naming it "my-first-branch" and then moving on the page reloaded successfully till the last step. In the end I was graded a score of 1 out of 4. My assignment is getting rejected.

After this, I have tried about 6 times after deleting and creating new repo, repeating the same procedure again but the bot wont refresh or grade my assignment anymore. Please help as this is also preventing me to moving on to next assignment of "Primer: Codespaces" .Kindly help.

**Comment by cmwilson21** [comment]
@Shahmeer-Nasir Thanks for opening a discussion! Other users have experienced that same issue when naming the branch, so I've updated the [Skills course](https://github.com/skills/introduction-to-github) text to clarify that the name has to be `my-first-branch`. 

As for the Autograde bot, it seems you are following this course from Education. Please open a discussion in the [Education Community](https://github.com/orgs/community/discussions/categories/github-education).
